packages/headless/src/modules/compare/services/index.ts

```diff
@@ -0,0 +1,138 @@
+import { HttpStatus, Injectable } from '@nestjs/common';
+import { Compare } from 'src/entity/compare';
+import { Helper } from 'src/helper/helper.interface';
+import { CompareResponse } from '../dto/compare.dto';
+import { CompareRepository } from '../repositories';
+
+@Injectable()
+export class CompareService {
+  constructor(
+    private compareRepository: CompareRepository,
+    private helper: Helper,
+  ) {}
+
+  async addItemToCompare(
+    userId: string,
+    productId: string,
+  ): Promise<CompareResponse> {
+    if (!productId.trim()) {
+      return this.helper.serviceResponse.errorResponse(
+        'Invalid product id.',
```

Comment:
Can you please add these error message as enum in models and then use them in swagger and here